content/python/concepts/functions/functions.md

```diff
@@ -15,62 +15,27 @@ CatalogContent:
 
 Some tasks need to be performed multiple times within a program. Rather than rewrite the same code in multiple places, a function may be defined using the `def` keyword. Function definitions may include parameters, providing data input to the function.
 
-Functions may return a value using the `return` keyword followed by the value to return.
+## Syntax
 
-```py
-# Define a function named my_function with parameter x
-
-def my_function(x):
-  return x + 1
-
-# Invoke the function and print the result
+```pseudo
+def my_function(value):
+  return value + 1
 
-print(my_function(2))      # Output: 3
-print(my_function(3 + 5))  # Output: 9
+print(my_function(2))
+print(my_function(3 + 5))
```

Comment:
Shouldn't we retain the output in some form?

Reply:
That's fair. I'll put it back.